### DISCUSSION TITLE
C'est quoi le Control Flow ? | Bonjour Angular

### DISCUSSION THREAD

**Discussion by giscus** [bonjour-angular/apprendre#9] [General]
# C'est quoi le Control Flow ? | Bonjour Angular

C'est quoi le Control Flow dans Angular ?

https://apprendre.bonjour-angular.com/cest-quoi/control-flow/

**Comment by jnr1507** [comment]
vraiment claire l'explication

**Comment by hamtaroratmah** [comment]
Explications très claire ! Seul petit soucis pour la dernière phrase du @if où il est écrit @for à la place 😊

**Reply by KevTale** [reply]
C'est fixed!

**Comment by zyrass** [comment]
Merci pour les explications, c'est limpide !

**Comment by nabil-kb** [comment]
Très bien détaillé je vous remercie Kevin pour les explications, est ce que je peux avoir une version pdf svp ?
